lib/cdo/config.rb

```diff
@@ -1,4 +1,5 @@
 require 'singleton'
+require 'delegate'
```

Comment:
I had to add this line after removing `require 'aws/google'` from `cdo_google_credentials`, it might have been implicitly required somewhere in `aws-google` that now needed to be made explicit.